tasks/39_unique-in-order/unique-in-order.js

```diff
@@ -0,0 +1,11 @@
+function uniqueInOrder(arr) {
+  const result = [];
+  for (let i = 0; i < arr.length; i++) {
+    if (arr[i] !== arr[i+1]) {
```

Comment:
The same here. You are going behind of array borders

Reply:
Binary operators should go with spaces around like `arr[i + 1]`

Reply:
fixed it